examples/analog_in/cont_voltage_acq_int_clk.py

```diff
@@ -0,0 +1,31 @@
+﻿"""Example of analog input voltage acquisition.
+
+This example demonstrates how to acquire a continuous amount of data
+using the DAQ device's internal clock.
+"""
+
+import pprint
+import time
+
+import nidaqmx
+from nidaqmx.constants import AcquisitionType
+
+pp = pprint.PrettyPrinter(indent=4, compact=True)
+
+with nidaqmx.Task() as task:
+    task.ai_channels.add_ai_voltage_chan("Dev1/ai0")
+    task.timing.cfg_samp_clk_timing(
+        1000.0, sample_mode=AcquisitionType.CONTINUOUS, samps_per_chan=1000
+    )
+    task.start()
+
+    try:
+        while True:
+            data = task.read(number_of_samples_per_channel=100)
+            print(f"Acquired data: {pp.pformat(data)}")
+            print("Press Ctrl+C to stop.")
+            time.sleep(0.5)
```

Comment:
This example acquires 1000 samples per second and reads 100 samples at a time, so it must read 10 times a second in order to keep up with the samples that it acquires.

`task.read()` already waits for the data to arrive (when using a physical device). Calling `time.sleep()` is unnecessary and sleeping for more than 1/10th of a second causes this example to fall behind. After a minute or so, this leads to an overflow error:

```
Traceback (most recent call last):
  File "D:\dev\nidaqmx-python\examples\analog_in\cont_voltage_acq_int_clk.py", line 24, in <module>
    data = task.read(number_of_samples_per_channel=100)
  File "D:\dev\nidaqmx-python\generated\nidaqmx\task\_task.py", line 615, in read
    _, samples_read = self._interpreter.read_analog_f64(
  File "D:\dev\nidaqmx-python\generated\nidaqmx\_library_interpreter.py", line 4149, in read_analog_f64
    self.check_for_error(error_code, samps_per_chan_read=samps_per_chan_read.value)
  File "D:\dev\nidaqmx-python\generated\nidaqmx\_library_interpreter.py", line 6377, in check_for_error
    raise DaqReadError(extended_error_info, error_code, samps_per_chan_read)
nidaqmx.errors.DaqReadError: The application is not able to keep up with the hardware acquisition.
Increasing the buffer size, reading the data more frequently, or specifying a fixed number of samples to read instead of reading all available samples might correct the problem.

Property: DAQmx_Read_RelativeTo
Corresponding Value: DAQmx_Val_CurrReadPos
Property: DAQmx_Read_Offset
Corresponding Value: 0

Task Name: _unnamedTask<0>

Status Code: -200279
```

To slow down the screen updates, you must increase the ratio between samples to read and the sample rate. You can do this by increasing samples to read or decreasing the sample rate.

Reply:
The intention of `time.sleep()` is to show the "Press Ctrl+C to stop." to remind user on every loop. I will refer `cont_voltage_acq_int_clk_every_n_samples_event.py` to show the message before task starts.